packages/driver/cypress/e2e/commands/actions/check.cy.js

```diff
@@ -803,58 +824,6 @@ describe('src/cy/commands/actions/check', () => {
   })
 
   context('#uncheck', () => {
-    it('does not change the subject', () => {
```

Comment:
These test removals are not necessary for detached DOM; but looking through the file, the tests for `.check()` and `.uncheck()` are extremely redundant, considering they share an implementation.

```
Commands.addAll({ prevSubject: 'element' }, {
    check (subject, values, options) {
      return checkOrUncheck.call(this, Cypress, cy, 'check', subject, values, options)
    },

    uncheck (subject, values, options) {
      return checkOrUncheck.call(this, Cypress, cy, 'uncheck', subject, values, options)
    },
  })
```

That is, they invoke the same function under the hood, sharing 90% of the logic and varying only in very specific ways, eg:

```
  const isNoop = ($el) => {
    return type === 'check'
      ? $el.prop('checked')
      : !$el.prop('checked')
  }
```

But we test the same features of both commands, eg:

```
  context('#check', () => {
    it('does not change the subject', () => {...})
    it('changes the subject if specific value passed to check', () => {...})

    it('can check a checkbox or radio', () => {...})
  })

  context('#uncheck', () => {
    it('does not change the subject', () => {...})
    it('changes the subject if specific value passed to uncheck', () => {...})

    it('can uncheck a checkbox', () => {...})
    it('throws specifically on a radio', () => {...})
  })
```

So we're basically running the same tests twice. There are certainly some tests that are different - eg, you can check a checkbox or radio, but you cannot uncheck a radio. But I don't see much value in testing the same code multiple times for the parts they do share, and spent a few minutes removing some of the duplicates in this file.

The same can be said of several other commands, but I didn't want to go through too many files and do the same thing unless other people agree with the approach.

I tried to ask questions about this on slack, but didn't get much engagement, so here it is in PR form.

Reply:
Yeah, but these tests aren't duplicates. They cover 95% of the same stuff but 5% is different, so I don't think it's a good idea to remove them completely. It doesn't look like the changes were needed for this PR, so can you revert them for now until we have a less duplicative way of testing `.uncheck`?

Reply:
Yeah, please do. I also dislike the fact that this wastes time the way it is but we can't remove the canonical tests for a command without guarantees that it's still being tested (code coverage when?)

Reply:
Restored tests in commit 22b10f409934c43228ab96e0addde287245364be.

Reply:
It was not "all .uncheck() tests", it was "this test, and this test, and this other test" - I looked at the tests and the code they executed, and compared them one-by-one to the `.check()` tests, which is the same guarantee that we get with any other code change. :woman_shrugging:

Reply:
I'm not as familiar with these tests as you. But since we do switch some behavior between `check` and `uncheck`,  even though these tests still exist in `context('#check'`, it's not clear to me that we are actually not losing any coverage by removing them, given my limited powers of brain-powered static analysis. Since this PR isn't held back by having these tests, I figured we could just as well keep them for now, to avoid having to review 400 lines of removed tests in an unrelated PR.

I love deleting excess tests, though. Given that you feel confident we aren't losing any coverage here, feel free to request me on a follow-up PR to remove these. I'd be happy to give it a closer look instead of just delivering my immediate reaction. But I don't want to block this PR any longer with this.